llmfoundry/command_utils/eval.py

```diff
@@ -65,6 +68,17 @@ def evaluate_model(
     should_log_config: bool = True,
     load_path: Optional[str] = None,
 ):
+    if fsdp_config:
+        warnings.warn(
+            VersionedDeprecationWarning(
+                'The argument fsdp_config is deprecated. Please use parallelism_config instead.',
+                remove_version='0.12.0',
```

Comment:
what's our policy on this? should this be a later version?

Reply:
yes, +2 versions